crates/sui-types/src/unit_tests/messages_tests.rs

```diff
@@ -435,3 +437,14 @@ fn test_digest_caching() {
     // cached digest was not serialized/deserialized
     assert_ne!(initial_effects_digest, *deserialized_effects.digest());
 }
+
+#[test]
+fn test_digest_caching_11() {
```

Comment:
let's pick a better name for this test function

Reply:
Apologies again, this is not a test - it's a generator for valid pk + sig, so that we can write valid move tests.